ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -311,6 +311,41 @@ def test_tensorflow_Cos(  # NOQA
         x=x[0],
     )
 
+# Cross
+@handle_frontend_test(
+        fn_tree='tensorflow.raw_ops.Cross',
+        dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=3,
+        max_dim_size=3,
+        num_arrays=2,
+        shared_dtype=True,
```

Comment:
The CI test is failing due to high numbers overflowing to inf. This might be resolved if you set the max value or safety factor here.

Reply:
I add the **safety_factor_scale='log'** you can check it now

Reply:
I also add the    **max_value=1e5**